client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -591,6 +601,7 @@ const PluginSingleListView = ( {
 	}
 	return (
 		<PluginsBrowserList
+			activePlugins={ activePlugins }
```

Comment:
You'll need to add the same to `FullListView` which handles category pages like `/plugins/paid/:site`.

Reply:
good catch!